email-generator/app/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "email-generator",
+  "version": "1.0.0",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js"
+  },
+  "dependencies": {
```

Comment:
All versions should be locked to specific versions to prevent students running into compatibility issues in the future. prefer to use specific numbered versions of all deps